packages/core/src/utilities/word-break/word-break.example.html

```diff
@@ -0,0 +1,12 @@
+<section class="ds-l-container preview__grid">
+  <div class="ds-l-row">
+    <div class="ds-l-col--3 ds-u-word-break">
```

Comment:
I think changing the block from `ds-l-col--3` to `ds-l-col--2` better shows how the word break is working.
Maybe also consider showing them on separate lines?